sdk/communication/Azure.Communication.JobRouter/src/autorest.md

```diff
@@ -0,0 +1,22 @@
+﻿# Azure.Communication.JobRouter
+
+When a new version of the swagger needs to be updated:
+1. Go to sdk\communication\Azure.Communication.JobRouter\src, and run `dotnet msbuild /t:GenerateCode` to generate code.
+2. Upload the Azure.Communication.JobRouter.dll to the apiview.dev tool.
+If any of the new objects needs to be overwritten, add the required changes to the 'Models' folder.
+
+3. Repeat 2 and 3 until the decided interface is reflected in the apiview.dev 
+
+> see [https://aka.ms/autorest](https://aka.ms/autorest)
+
+## Configuration
+
+```yaml
+tag: package-jobrouter-2022-07-18-preview
+model-namespace: false
```

Comment:
Since we have a lot of models in this library, can we generate all the models in `Azure.Communication.JobRouter.Models` namespace?

If you remove `model-namespace: false` and regenerate code again, then all the models will be generated in `Azure.Communication.JobRouter.Models` namespace.

Reply:
The design implemeted here is that the subset of the models (everything used as input) should live under `Azure.Communication.JobRouter` and another subset lives under `Azure.Communication.Models` (everything used as output). The number of input models is greater than output models hence  `model-namespace: false`